### ISSUE TITLE
choosenim "branchname +Nim#7757"  to build against PR(s)

### ISSUE THREAD

**Issue by timotheecour** [dom96/choosenim#58]
in D, [Digger](https://github.com/CyberShadow/Digger) allows to build Nim against a specific PR, eg:
```
digger build "master +dmd#7757  +tools#292" # build master modified by 2 PR's against dmd and tools
# etc see docs
```

this is very useful for letting people try out PR's before they're pushed with minimal effort.

in choosenim this could be:

```
choosenim "branchname +Nim#7757" 
```

**Comment by dom96** [comment]
Not everything needs to be an RFC :)

**Comment by genotrance** [comment]
Similar to #55.

If this can issue can be merged with that and discussed together for a common design, it will be helpful.

**Comment by dom96** [comment]
Yeah, this should be resolved via #55.
